frontend/src/components/messages/NewRequestDialog.vue

```diff
@@ -185,17 +389,38 @@ export default {
         this.setUpDefaultNewRequestModel()
       },
     },
+    'newRequestModel.subCategories': {
+      handler() {
+        if (this.changeTypeClass === 'change-type-required' && this.isAnySubCategoryChecked) {
+          this.changeTypeClass = ''
+        }
+      },
+      deep: true
+    }
   },
   created() {
+    this.PHONE_FORMAT = PHONE_FORMAT
+    this.EMAIL_FORMAT = EMAIL_FORMAT
+    this.AUTO_REPLY_MESSAGE = 'Your change request is complete.'
+    this.requestSubCategorySortOrder = [
+      "Organization Details",
+      "Facility Details",
+      "Organization phone/email",
+      "Facility phone/email",
+      "Add/change a licence",
+      "Other"
+    ]
+    this.sortedRequestSubCategories = this.sortRequestSubCategories(this.requestSubCategories, this.requestSubCategorySortOrder)
     this.setUpDefaultNewRequestModel()
   },
   methods: {
-    ...mapActions(useMessagesStore, ['createAssistanceRequest', 'addNewAssistanceRequestToStore']),
+    ...mapActions(useMessagesStore, ['createAssistanceRequest', 'addNewAssistanceRequestToStore', 'updateAssistanceRequest', 'updateAssistanceRequestInStore', 'replyToAssistanceRequest', 'getAssistanceRequest', 'getAssistanceRequests']),
 
     setUpDefaultNewRequestModel() {
       this.newRequestModel = {
+        requestCategoryId: REQUEST_CATEGORY_TYPES.ACCOUNT_MAINTENANCE,
```

Comment:
Is there a business requirement that we always default the request category to be Account Maintenance even when the user create the new request for Messages page?

Reply:
No, oversight... fixed.